README.md

```diff
@@ -33,24 +33,24 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 [Learn more 👉](https://www.gitpod.io/features/)
 
-Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free subscription for open-source or a [free self-hosted version](https://www.gitpod.io/self-hosted/). An enterprise license is available [here](https://www.gitpod.io/self-hosted/).
+Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free subscription for open-source or a [free self-hosted version](https://www.gitpod.io/self-hosted). An enterprise license is available [here](https://www.gitpod.io/self-hosted).
```

Comment:
**Minor**: The canonical link here still seems to have a final `/`:

```suggestion
Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free subscription for open-source or a [free self-hosted version](https://www.gitpod.io/self-hosted/). An enterprise license is available [here](https://www.gitpod.io/self-hosted/).
```

Reply:
Got it.